File: BE/DTOs/AnswerDTO.cs
namespace BE.DTOs
{
    public class AnswerDTO
    {
        public string Id { get; set; } = "";
        public string Content { get; set; } = "";
        public bool IsCorrect { get; set; } = false;
        public bool IsSelected { get; set; } = false;
        public AnswerDTO() { }
        public AnswerDTO(string id, string content, bool isCorrect, bool isSelected)
        {
            Id = id;
            Content = content;
            IsCorrect = isCorrect;
            IsSelected = isSelected;
        }
    }
}

File: FE/app/src/main/java/com/example/tqt_quiz/domain/dto/AnswerDTO.java
package com.example.tqt_quiz.domain.dto;

import com.google.gson.annotations.SerializedName;

public class AnswerDTO {
    @SerializedName("id")
    String id;
    @SerializedName("content")
    String content;
    @SerializedName("isCorrect")
    boolean isCorrect;
    @SerializedName("isSelected")
    boolean isSelected;
    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getContent() {
        return content;
    }

    public void setContent(String content) {
        this.content = content;
    }

    public boolean isCorrect() {
        return isCorrect;
    }

    public void setCorrect(boolean isCorrect) {
        this.isCorrect = isCorrect;
    }

    public boolean isSelected() {
        return isSelected;
    }

    public void setSelected(boolean isSelected) {
        this.isSelected = isSelected;
    }
}
